### ISSUE TITLE
v5.0.33a (Windows): Miner closes without error message

### ISSUE THREAD

**Issue by UselessGuru** [rplant8/cpuminer-opt-rplant#45]
RplantCpu-v5.0.33a\cpuminer-Avx2.exe --algo yespowerSugar --url stratum+tcp://yespowerSUGAR.eu.mine.zpool.ca:6241 --user *************************** --pass ******************,c=BTC --cpu-affinity AAAA --quiet --threads 5 --api-bind=4000 --debug

```
         **********  cpuminer-rplant 5.0.33aW-avx ***********
CPU: Intel(R) Core(TM) i5-8600K CPU @ 3.60GHz VM
SW built on Jul  3 2023 with GCC 12.1.1
CPU features:  AVX2    AES
SW features:   AVX     AES
Algo features: SSE2   SHA

Starting miner with SSE2...

[2023-07-04 10:59:26] CPU affinity [......]
[2023-07-04 10:59:26] Transparent Huge Pages +
[2023-07-04 10:59:26] Creating stratum thread
[2023-07-04 10:59:26] Creating API thread
[2023-07-04 10:59:26] Stratum connect yespowerSUGAR.eu.mine.zpool.ca:6241
[2023-07-04 10:59:26] API listnening to 127.0.0.1:4000
[2023-07-04 10:59:26] Miner thread priority 0 (nice 19)
[2023-07-04 10:59:26] affine_to_cpu_mask for 0 returned 57
[2023-07-04 10:59:26] Binding thread 0 to mask 0000000000000000
[2023-07-04 10:59:26] affine_to_cpu_mask for 1 returned 57
[2023-07-04 10:59:26] Binding thread 1 to mask 0000000000000000
[2023-07-04 10:59:26] Stratum session id: 41481cca1b8a62ce2d1c0b7c20280556
[2023-07-04 10:59:26] Stratum extranonce1= feff0078, extranonce2 size= 4
[2023-07-04 10:59:26] affine_to_cpu_mask for 2 returned 57
[2023-07-04 10:59:26] Binding thread 2 to mask 0000000000000000
[2023-07-04 10:59:26] affine_to_cpu_mask for 3 returned 57
[2023-07-04 10:59:26] Binding thread 3 to mask 0000000000000000
[2023-07-04 10:59:26] 5 of 6 miner threads started using 'yespowerSUGAR' algorithm
[2023-07-04 10:59:26] affine_to_cpu_mask for 4 returned 57
[2023-07-04 10:59:26] Binding thread 4 to mask 0000000000000000
[2023-07-04 10:59:26] Extranonce subscription enabled
[2023-07-04 10:59:26] Threads restarted for new work.
[2023-07-04 10:59:26] Stratum connection established
[2023-07-04 10:59:26] Threads restarted for new work.
[2023-07-04 10:59:26] New stratum difficulty 0.01
[2023-07-04 10:59:26] yespowerSUGAR.eu.mine.zpool.ca:6241 yespowerSUGAR block 24265901, job 1775, network diff 0.00041937904
                      Diff: Net 0.00041938, Stratum 0.01, Target 1.5259e-07

c:\**************\NemosMiner>
```

This happens on different pools (ZPool, ZergPool) and different algorithms, e.g. Allium, Ghostrider, Mike, MinotaurX, Yescrypt, YescryptR16, YescryptR32, Yespower, YespowerR16, YespowerSugar

**Comment by RainbowMiner** [comment]
I can confirm this issue - happening on all my machines.

**Comment by UselessGuru** [comment]
Fixed with 5.0.34
